### DISCUSSION TITLE
Failed to construct 'Worker'

### DISCUSSION THREAD

**Discussion by liangchabuliang** [epam/ketcher#803] [Q&A]
My project is Vue，so,I used like this  						<iframe id="ketcherIframe" ref="ketcherIframe"  name="ketcherIframe" frameborder="0" :src="`${publicPath}/standalone/index.html`" width="700" height="400"></iframe>  I used in two template .  when i close one of them ,then Jump to another template  by route ，The console appears ’main.3d47a386.chunk.js:1 Uncaught (in promise) DOMException: Failed to construct 'Worker': The context provided is invalid.‘ there is another case,one template   include two ketcher,these problems also occur

**Comment by AndreiMazol** [comment]
Hi @liangchabuliang ,

 could you provide ready-to-run example that covers your case? if it is not possible perhaps you can share the video with your scenario (domain + network tab should be visible).

Thank you in advance.
